app/routes/auth.route.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
```

Comment:
Codacy has a fix for the issue: [Replace `'express'` with `"express"`](https://app.codacy.com/gh/Eight-Corner/oneuleun_web_api/pullRequest?prid=9684192)
```suggestion
const express = require("express");
```